src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/ui-data/lambda-metrics.mdx

```diff
@@ -0,0 +1,385 @@
+            <td>
+            Function
+            </td>
+            <td>
+            </td>
+            <td>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Last modified
+            </td>
+            <td>
+            Last modified time of a function.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Cold start rate
+            </td>
+            <td>
+            Percentage of invocations that were cold started, correlation with services.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Cold start count
+            </td>
+            <td>
+            The number of times a function encountered a cold start.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Cold start count
+            </td>
+            <td>
+            Avg, min, and max time it takes for a function to start after being idle.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Estimated costs
+            </td>
+            <td>
+            Estimated cost of running a function.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Retries
+            </td>
+            <td>
+            Number of times a function was retired because it failed
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Failures
+            </td>
+            <td>
+            Number of times a function failed.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Billed duration
+            </td>
+            <td>
+            NThe time of function execution that was billed.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Billed network bytes
+            </td>
+            <td>
+            Total network bytes that a function generated.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Async events dropped
+            </td>
+            <td>
+            The number of events dropped without successfully running the function.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Async events received
+            </td>
+            <td>
+            The number of events received by the function.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Async event age
+            </td>
+            <td>
+            AsyncEventAge is a measure of time between Lambda successfully queuing the event and invoking the function.
```

Comment:
@cbehera-newrelic `AsyncEventAge` looks like the actual metric name you'd query in the query builder. Could you add another column that has all of the exact names? Or, you could replace the values in the column for the common names with the literal names of the metrics. What do you think?

Reply:
It's good idea to replace metrics column with exact metrics names. Creating one more column might confuse users.